src/Executables/ConvertComposeTable/ConvertComposeTable.cpp

```diff
@@ -46,17 +47,88 @@ void convert_file(const std::string& compose_directory,
 int main(int argc, char** argv) {
   namespace bpo = boost::program_options;
   try {
-    bpo::options_description command_line_options;
+    bpo::options_description command_line_options(
+        "This executable converts an ASCII formatted CompOSE 3d equation of "
```

Comment:
Question: Would these steps also be good as a user guide? Like we have for CCE and processing SpEC data?

Reply:
I thought about this and I wasn't sure... I decided to put it here so it's with the executable that handles the conversion :shrug:

Reply:
Opened an issue #5419.